src/redux/reducers/store.js

```diff
@@ -4,21 +4,30 @@ import reduxThunk from 'redux-thunk';
 import logger from 'redux-logger';
 import rootReducer from './index';
 
+export const STORE_VERSION = '1';
+
 const saveState = (state) => {
   try {
     localStorage.setItem('state', JSON.stringify(state));
+    localStorage.setItem('storeVersion', STORE_VERSION);
```

Comment:
What happens if you change 'storeVersion' to some thing else? In how many places would you have to change it?

Reply:
Making this a constant.